lib/ldap.c

```diff
@@ -182,6 +182,41 @@ const struct Curl_handler Curl_handler_ldaps = {
 };
 #endif
 
+#ifdef USE_WIN32_LDAP
+static int Win_ldap_bind(struct connectdata *conn, LDAP *server, TCHAR *user, TCHAR *passwd)
+{
+    int rc = 0;
+    ULONG method = LDAP_AUTH_NEGOTIATE;
+    SEC_WINNT_AUTH_IDENTITY cred = { 0 };
+    PCHAR inuser = NULL;
+    PCHAR inpass = NULL;
+
+    if (user != NULL && passwd != NULL) {
+        cred.User = user;
+        cred.UserLength = strlen(user);
+        cred.Password = passwd;
+        cred.PasswordLength = strlen(passwd);
+        cred.Flags = SEC_WINNT_AUTH_IDENTITY_ANSI;
```

Comment:
I use Steve's wrapper, it works if perl is in your path. Open a command window at the projects directory and run checksrc. It's a checksrc.bat that wraps the checksrc.pl.